content/en/02-installing-pixie/03-install-guides/03-airgap-pixie.md

```diff
@@ -0,0 +1,281 @@
+
+```bash
+kubectl get pods -n plc
+```
+
+> Note about **Step 10**: if you applied the `cloud.yaml` manifest multiple times (for example, while resolving an ImagePullError), you will see the following `create-admin-job` and `create-hydra-client-job` errors:
+
+```
+create-admin-job time="2022-07-05T21:27:40Z" level=fatal msg="Org 'default' with domain 'default.com' already exists. Remove the org from the database or change the org name."
+```
+
+```
+create-hydra-client-job {
+create-hydra-client-job   "error": "Unable to insert or update resource because a resource with that value exists already"
+create-hydra-client-job }
+```
+
+> If you get these errors, delete the `plc` namespace and start over from **Step 4** of the install guide.
+
+### Set up DNS
+
+1. Ensure that the `cloud-proxy-service` and `vzconn-service` LoadBalancer services have External IPs assigned. If you are running Pixie Cloud on `minikube`, you likely need to run `minikube tunnel` before continuing with this setup.
+
+```bash
+# If using minikube, run:
+minikube tunnel
+
+# Check if external IPs have been assigned
+kubectl get service cloud-proxy-service -n plc
+kubectl get service vzconn-service -n plc
+```
+
+2. Setup your DNS. This produces a `dev_dns_updater` binary in the top level `pixie` directory.
+
+```bash
+go build src/utils/dev_dns_updater/dev_dns_updater.go
+```
+
+3. You'll need to hardcode in your kube config. Leave this tab open.
+
+```bash
+./dev_dns_updater --domain-name="dev.withpixie.dev"  --kubeconfig=$HOME/.kube/config --n=plc
+```
+
+4. Navigate to `dev.withpixie.dev` in your browser. Make sure that the network you are on can access your cluster.
+
+### Authentication using Kratos / Hydra
+
+Self-managed Pixie Cloud only supports one organization.
+
+1. To setup the default admin account, check the logs for the `create-admin-job` pod by running:
+
+```bash
+kubectl logs create-admin-job-<pod_string> -n plc
+```
+
+2. Open the URL from the pod's logs to set the password for the `admin@default.com` user.
+
+<Alert variant="outlined" severity="warning">
+  If you've visited dev.withpixie.dev before, make sure to clear the cookies for this site or you'll get a login error.
+</Alert>
+
+3. Once the password has been set, login using `admin@default.com` for the `identifier` and your new password.
+
+<Alert variant="outlined" severity="warning">
+  There is a known issue with login on self-managed Pixie Cloud on Safari and Firefox. For now, use Chrome.
+</Alert>
+
+### Serve the Script Bundle
+
+After logging into the Pixie UI using the admin account, you will see a blank UI with the following errors in the DevTools console:
+
+<svg title='' src='airgap/ui-error.png' />
+
+To resolve these errors, you'll need to do one of the following:
```

Comment:
I would just choose one method in the following. It seems like it would be unclear to the user which method they would choose.